README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### A Little Bit About Me!
+
+My name is Lauren Park. I am a second year at the College, and am a Sociology and French major. I was born and raised in Los Angeles. I am also on the women's varsity tennis team. 
+
+#### These are a few of *my favorite things* to do in Chicago:
+* Eating at new restaurants
+* Running along the lake
+* Exploring new neighborhoods
+* Sitting on the quad when the weather is nice
+
+[Here is the link to one of my favorite food blog guides! Enjoy!:)](https://chicago.eater.com/2018/1/5/16151978/best-food-chicago-restaurants-city-guide)
```

Comment:
It might make more sense to just hyperlink the [Here] section of that sentence, instead of making the whole section a hyperlink.